src/main/kotlin/carracing/Car.kt

```diff
@@ -0,0 +1,17 @@
+package carracing
+
+import kotlin.random.Random
+
+class Car {
+    private var position: Int = 0
+
+    fun move() {
+        if (Random.nextInt(10) >= 4) {
+            position++
+        }
+    }
```

Comment:
If your main logic includes something hard to test, like randomness, how can you make it testable without mockkObject?